packages/web-components/src/components/va-text-input/va-text-input.tsx

```diff
@@ -229,12 +229,12 @@ export class VaTextInput {
           part="input"
         />
         {maxlength && value?.length >= maxlength && (
-          <small aria-live="polite" part="validation">
+          <small part="validation">
```

Comment:
We're modifying this component because it is used in the date component:

https://github.com/department-of-veterans-affairs/component-library/blob/36614239b1ca1984ebe58a306c11860ab9178e6b/packages/web-components/src/components/va-date/va-date.tsx#L257-L272